bin/common/utils.js

```diff
@@ -49,6 +48,15 @@ exports.loadNuxtConfig = (argv) => {
     if (options.default) {
       options = options.default
     }
+
+    if (typeof options === 'function') {
+      try {
+        options = await options()
```

Comment:
Usage of top-level `async/await` will break `nuxt-legacy` for Node 6 support. We have to decide to deprecate that package or use normal `Promise`.

Reply:
Alternative idea: Use webpack to transpile `nuxt-legacy/bin` scripts :D

Reply:
Though all major cloud platforms support Node 8, I wouldn't like to see `nuxt-legacy` deprecated yet. So either Promises (which hurt the readability a little) or webpack (which introduces another build step, but only for the core-team and not for the users).

Webpack seems to be the better choice, doesn't it?

Reply:
Using babel or babylon to this is as simple as something like this:

```
yarn add --dev babel-cli
yarn babel bin -x ".js," -d packages/nuxt-legacy/bin
```